cmd/cosign/cli/options/verify.go

```diff
@@ -108,6 +109,9 @@ func (o *VerifyAttestationOptions) AddFlags(cmd *cobra.Command) {
 
 	cmd.Flags().StringVarP(&o.Output, "output", "o", "json",
 		"output format for the signing image information (json|text)")
+
+	cmd.Flags().BoolVar(&o.LocalImage, "local-image", false,
+		"whether the path to the image is a local directory")
```

Comment:
wdyt of clarifying that the image has to be saved with cosign, something like

```
"path to an image saved locally via `cosign save`"
```

Reply:
this would match the description below as well! we might want to update the description for the flag in `cosign verify` as well

Reply:
Thanks, updated!